styles.css

```diff
@@ -357,7 +398,17 @@ body {
 }
 
 #directorHeading {
-color: #cccccc
+  color: #cccccc
```

Comment:
for color codes, if all the letters are the same, we can just use three letters instead of six. e.g `#ccc`